backend/src/main/resources/schema.graphqls

```diff
@@ -188,31 +195,39 @@ type PatientLink {
 }
 
 type Query {
-  deviceType: [DeviceType]
+  deviceType: [DeviceType] # TODO what permission is required here?
```

Comment:
The device type doesn't contain any information that isn't also public info. My vote is this is open to any authenticated user. 

Currently I believe this is only used by admins in facility form. However we have plans to call this query from the test results. If we have to lock it down we should be able to ban entry one users from accessing this query

Reply:
Basically everywhere except the facility-admin form, we should be (and I think are?) using the `whoami` output.